uploader/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.1 on 2020-03-07 14:04
```

Comment:
Can you delete this migration and make sure that in this PR, we only have the stuff to add the new model as we want it ?

Reply:
Sorry my bad! Will fix it right away!